src/utils/routePathsConfig.tsx

```diff
@@ -0,0 +1,221 @@
+import OverView from "../pages/OverView";
+import AuditSnapshot from "../pages/confidentialVm/AuditSnapshot";
+import DeployApplication from "../pages/confidentialVm/DeployApplication";
+import DetectMalware from "../pages/confidentialVm/DetectMalware";
+import SnapshotVm from "../pages/confidentialVm/SnapshotVm";
+import UseConfidentialVm from "../pages/confidentialVm/UseConfidentialVm";
+import VerifyIntegrity from "../pages/confidentialVm/VerifyIntegrity";
+import AboutCovercrypt from "../pages/covercrypt/AboutCovercrypt";
+import CreateEncryptionPolicy from "../pages/covercrypt/CreateEncryptionPolicy";
+import CreateMasterKeyPair from "../pages/covercrypt/CreateMasterKeyPair";
+import DecryptData from "../pages/covercrypt/DecryptData";
+import EncryptData from "../pages/covercrypt/EncryptData";
+import GenerateUDK from "../pages/covercrypt/GenerateUDK";
+import SetupCovercrypt from "../pages/covercrypt/SetupCovercrypt";
+import AboutCse from "../pages/cse/AboutCse";
+import SetupCse from "../pages/cse/SetupCse";
+import SummarizeDocument from "../pages/cse/SummarizeDocument";
+import AboutFindex from "../pages/findex/AboutFindex";
+import IndexDatabase from "../pages/findex/IndexDatabase";
+import InstantiateFindex from "../pages/findex/InstantiateFindex";
+import SearchInDatabase from "../pages/findex/SearchInDatabase";
+import SetupFindex from "../pages/findex/SetupFindex";
+import AboutPKI from "../pages/pki/AboutPKI";
+import DecryptDataPKI from "../pages/pki/DecryptDataPKI";
+import EncryptDataPki from "../pages/pki/EncryptDataPki";
+import GetCertificate from "../pages/pki/GetCertificate";
+import GrantAccess from "../pages/pki/GrantAccess";
+import ImportAndUnwrapUDK from "../pages/pki/ImportAndUnwrapUDK";
+import SendWrappedDecryptionKey from "../pages/pki/SendWrappedDecryptionKey";
+import SetupPki from "../pages/pki/SetupPki";
+import UploadCert from "../pages/pki/UploadCert";
+
+export interface routePath {
+  path: string;
+  label?: string;
+  component?: JSX.Element;
```

Comment:
I am just hesitating bcs the more we add mandatory attribs the harder updates might become - I made it optionnal because in the existing code there was already some condition ( check router.tsx, `<>missing component</>` line ) that handles this so I said let's stay loyal to it ?

Reply:
Okaye for component, if you prefer to add your "place holder" - but label should be required

Reply:
Done :D